design.html

```diff
@@ -0,0 +1,1144 @@
+                          </div>
+                        </div>
+                      </div>
+                      <div class="container py-3 border-bottom">
+                        <div class="media  text-justify">
+                          <img src="./assets/things/middle9.PNG" class="align-self-center mr-3"  alt="John Doe" >
+                          <div class="media-body">
+                            <a class=" h5 font-weight-bold text-body" href="">Ahmaud Arbery's Mother Says George Floyd's Death 'Sent Me Back'</a><br>
+                            <span class="h6">To revive the economy, Joe Biden wants to build his own New Deal with an array of proposals to strengthen the social </span>      
+                          </div>
+                        </div>
+                      </div>
+                      <div class="container py-3 border-bottom">
+                        <div class="media  text-justify">
+                          <img src="./assets/things/middle10.PNG" class="align-self-center mr-3"  alt="John Doe" >
+                          <div class="media-body">
+                            <a class=" h5 font-weight-bold text-body" href="">Ahmaud Arbery's Mother Says George Floyd's Death 'Sent Me Back'</a><br>
+                            <span class="h6">To revive the economy, Joe Biden wants to build his own New Deal with an array of proposals to strengthen the social </span>      
+                          </div>
+                        </div>
+                      </div>
+                      <div class="container py-3 border-bottom">
+                        <div class="media  text-justify">
+                          <img src="./assets/things/middle11.PNG" class="align-self-center mr-3"  alt="John Doe" >
+                          <div class="media-body">
+                            <a class=" h5 font-weight-bold text-body" href="">Ahmaud Arbery's Mother Says George Floyd's Death 'Sent Me Back'</a><br>
+                            <span class="h6">To revive the economy, Joe Biden wants to build his own New Deal with an array of proposals to strengthen the social </span>      
+                          </div>
+                        </div>
+                      </div>
+                      <div class="container py-3 border-bottom">
+                        <div class="media  text-justify">
+                          <img src="./assets/things/middle12.PNG" class="align-self-center mr-3"  alt="John Doe" >
+                          <div class="media-body">
+                            <a class=" h5 font-weight-bold text-body" href="">Ahmaud Arbery's Mother Says George Floyd's Death 'Sent Me Back'</a><br>
+                            <span class="h6">To revive the economy, Joe Biden wants to build his own New Deal with an array of proposals to strengthen the social </span>      
```

Comment:
- [ ] In the More Stories section, the image of the article should not have full height, and the text should wrap around the image.

| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/87717304-8d656900-c786-11ea-9338-e82af48c3e20.png) | ![image](https://user-images.githubusercontent.com/61479648/87717251-7c1c5c80-c786-11ea-8346-e3fd7c77e0e6.png) |